scripts/build.js

```diff
@@ -0,0 +1,54 @@
+const concat = require('concat')
+    uglify = require('uglify-es'),
+    babel = require('@babel/core'),
+    fs = require('fs'),
+    path = require('path');
+
+const config = {
+    buildDirPath: path.dirname(__dirname) + '/build',
+    buildFileName: 'FrostFlake.min.js',
+    babelOptions: {
+        plugins: [
+            "@babel/plugin-proposal-class-properties",
+            "@babel/plugin-proposal-private-methods"
+        ]
+    },
+    uglifyOptions: {
+        sourceMap: process.argv[2] === "no-sourceMap" ? false : {url: "inline"}
+    }
+}
+
+concat([
+    './src/GameTime.js',
```

Comment:
Instead of specifying each file, can we glob everything in `src`? Otherwise I will definitely forget to add new files to this list!

Reply:
I am not 100% sure. I think, but I was worried about things being concatenated in the wrong order. Class definitions are hoisted which means that you should be able to access them before they are defined. However, I think that static properties breaks this for reasons I don't 100% understand.

The work around to this to to make each src file a module by exporting the class definition, then importing each class when needed.

Let me know which direction you would like to go.